### PR TITLE
Skip building tests and benchmarks

### PR DESCRIPTION
**Pull request by certik**

### CODE REVIEW

**Comment by isuruf** [comment]
We should build these tests in `spkg-check`

**Comment by certik** [comment]
I see. We can still skip the benchmarks at least.

And `spkg-check` should also test the Python tests.

File: spkg-install [hunk 1]
```diff
@@ -8,6 +8,8 @@ cmake -DCMAKE_INSTALL_PREFIX=$SAGE_LOCAL \
     -DBUILD_BENCHMARKS=no \
```
Comment (isuruf) [inline spkg-install:8]: Benchmarks are skipped already here.

**Comment by isuruf** [comment]
Yes, python tests should be added to `spkg-check`

**Comment by certik** [comment]
I see. So this PR is not needed at all. Thanks for the review.

**Pull request closed without merging**